src/legacy/core_plugins/kibana/public/index.scss

```diff
@@ -18,7 +18,6 @@
 
 // Visualize styles
 @import './visualize/index';
-@import './visualize_embeddable/index';
```

Comment:
You seem to now be missing the import of this index file from inside the legacy folder. So the styles won't be compiled.